spinalcordtoolbox/scripts/sct_maths.py

```diff
@@ -750,6 +750,16 @@ def main(argv: Sequence[str]):
         except SctMathsValueError as e:
             printv(f"ERROR: -{operation}: {e}", 1, 'error')
 
+    # Post-processing of the results
+    if final_type == 'image':
+        current_value.save(arguments.o, dtype=arguments.type)
+        display_viewer_syntax([arguments.o], verbose=verbose)
+    else:
+        assert final_type == 'metric'
+        img1, img2, metric, metric_full = current_value
+        compute_similarity(img1, img2, arguments.o, metric, metric_full, verbose)
```

Comment:
How about `metric_todo` instead of `metric` for the name of the tuple? I think it pretty much _has_ to happen in the post-processing part of the script, but it would be more explicit. Thanks for your comments about naming, it's important to get it right.